tasks/arrayToObject.js

```diff
@@ -0,0 +1,21 @@
+// get the old array, store it in json variable
+const json = require('../data/oldCardsArray.json');
+const fs = require('fs');
+
+// new object to write
+const newJson = {};
+
+// iterate through the array
+for (let i = 0; i < json.length; i += 1) {
```

Comment:
I think it would be best just to `toUpperCase()` all the ids for consistency unless that causes key conflicts.